Find a functionally equivalent code:com.android.connectivitymanagertest.ConnectivityManagerTestBase.removeConfiguredNetworksAndDisableWifi:COMMENT
<com.android.connectivitymanagertest.ConnectivityManagerTestBase: boolean removeConfiguredNetworksAndDisableWifi()>
Method Modifier: public      protected   
Method Modifier: Yes         No          
Method Updated:  No          Yes         
Comment:/**
 * Remove configured networks and disable wifi
 */

@@ -2,11 +2,11 @@
     if (!disconnectAP()) {
         return false;
     }
-    sleep(SHORT_TIMEOUT);
+    SystemClock.sleep(SHORT_TIMEOUT);
     if (!mWifiManager.setWifiEnabled(false)) {
         return false;
     }
-    sleep(SHORT_TIMEOUT);
+    SystemClock.sleep(SHORT_TIMEOUT);
     return true;
 }

